### PR TITLE
check that environment is clean

### PR DESCRIPTION
**Pull request by VladimirStepanov**
close https://github.com/testcontainers/testcontainers-go/issues/158

### CODE REVIEW

**Comment by VladimirStepanov** [comment]
@mdelapenya Hi! Could you check this solution?

**Comment by mdelapenya** [comment]
> @mdelapenya Hi! Could you check this solution?

I think this would work 👍 

On the other hand, I'm missing a contributing guide in the project, not in this PR of course. I'd add the usage of this shell script (and the one for linting) once we create that guide.

File: scripts/check_environment.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+
+containers=$(docker ps --format "{{.ID}}" | wc -l)
+
+if [ "$containers" -eq "0" ]; then
+   exit 0
+fi
+echo "environment is not clean"
```
Comment (mdelapenya) [inline scripts/check_environment.sh:9]: Could we list the number of containers that are still running? or a least of them?

Comment (VladimirStepanov) [reply scripts/check_environment.sh]: @mdelapenya done

File: scripts/check_environment.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+i=0
+while [ $i -ne 10 ]
```
Comment (mdelapenya) [inline scripts/check_environment.sh:4]: IIUC we are going to loop 10 times waiting for the containers to be removed, sleeping 2 secs on each loop, right?

**Review by mdelapenya — APPROVED** [review]
LGTM, thanks for this small but important contribution!

File: scripts/check_environment.sh (hunk 2 above)
Comment (VladimirStepanov) [reply scripts/check_environment.sh:4]: @mdelapenya Yes, you understood correctly. I’ve noticed that containers were really removed from ‘docker ps’ output after N seconds( in average in ten seconds).

That’s why I decided to wait 20 seconds for the result. The script checks the Docker output 10 times every 2 seconds to speed up the process . And if the output is not empty after the loop, the script returns an error.

**Comment by mdelapenya** [comment]
In https://github.com/testcontainers/testcontainers-go/runs/6905008920?check_suite_focus=true we see there is a container not stopped:
Screenshot 2022-06-16 at 07 33 59

Could you try to identify it and stopped in the test suite?

**Comment by VladimirStepanov** [comment]
@mdelapenya Fixed:)
Now we pass all tests successfully

**Comment by VladimirStepanov** [comment]
@mdelapenya Could we merge it?

**Pull request merged**